### PR TITLE
George/ports

### PR DESCRIPTION
**Pull request by palladia**

### CODE REVIEW

File: Unix/cli/cli_c.c [hunk 1]
```diff
@@ -2404,73 +2404,117 @@ static MI_Result GetCommandLineOptions(
...
+    {
+        if (!opts.hostname)
+        {
+            Ftprintf(sout, MI_T("omicli: --auth option specified, but not --hostname.\n"));
+            return MI_RESULT_FAILED;
+        }
+    }
+
+
+    if (opts.port == 0)
```
Comment (paulcallen) [inline Unix/cli/cli_c.c]: port is only relevant for remoting (hostname)

File: Unix/wsman/wsmanclient.c [hunk 2]
```diff
@@ -1020,15 +1015,6 @@ MI_Result WsmanClient_New_Connector(
                  goto finished;
             }
         }
-        else if (miresult == MI_RESULT_NO_SUCH_PROPERTY)
```
Comment (paulcallen) [inline Unix/wsman/wsmanclient.c]: we still need a default if one isn't present

File: Unix/cli/cli_c.c (hunk 1 above)
Comment (palladia) [inline Unix/cli/cli_c.c]: Fixed.

File: Unix/wsman/wsmanclient.c (hunk 2 above)
Comment (palladia) [inline Unix/wsman/wsmanclient.c]: Fixed.

File: Unix/common/MI.h [hunk 3]
```diff
@@ -8690,9 +8690,9 @@ MI_INLINE MI_Result MI_DestinationOptions_GetPrivateKeyFile(
 **
 **=============================================================================
 */
-/*Transports vary depending on protocol.  For WSMAN, HTTP and HTTPS are supported */
+/*Transports vary depending on protocol.  For WSMAN, HTTP, HTTPS, and NONE are supported */
```
Comment (paulcallen) [inline Unix/common/MI.h]: NONE is not a valid transport

File: Unix/wsman/wsmanclient.c [hunk 4]
```diff
@@ -1004,41 +1004,37 @@ MI_Result WsmanClient_New_Connector(
...
-        {
-            /* Set defaults for a few things. */
-            secure = MI_FALSE;
 
-            self->wsmanSoapHeaders.protocol = MI_T("http");
+        miresult = MI_DestinationOptions_GetPacketPrivacy(options, &secure);
+        if (miresult != MI_RESULT_OK)
+        {
+            goto finished;
```
Comment (paulcallen) [inline Unix/wsman/wsmanclient.c]: need a default if one is not present

**Pull request merged**